src/mpi/romio/configure.ac

```diff
@@ -784,7 +784,7 @@ AC_ARG_WITH([cart],
 
 AS_IF([test "x$with_cart" != xno], [
     CART="yes"
-    LDFLAGS="$LDFLAGS -L$with_cart/lib"
+    LDFLAGS="$LDFLAGS -L$with_cart/lib -L$with_cart/lib64"
```

Comment:
This entire thing should be replaced with the `PAC_SET_HEADER_LIB_PATH` macros, for consistency.

Reply:
> @mchaarawi Your name isn't listed on the Intel CLA, so we cannot merge this PR till that is fixed. Intel is working on it.
> 
> Tagging @wesbland and @ddurnov too.

ok let me rework these changes and repush a squashed PR when I'm added to the CLA. I know the process was started.